### PR TITLE
Add script for bot matches

### PR DESCRIPTION
**Pull request by David-Cha**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -35,7 +35,10 @@ fn main() {
 
     if your_color == Color::White {
         let m = match generate_move(&game, Color::White) {
-            Some(m) => m,
+            Some(m) => {
+                println!("{}", m);
```
Comment (vdoubleu) [inline src/main.rs:39]: do we need this println here?

File: src/main.rs [hunk 2]
```diff
@@ -48,10 +51,10 @@ fn main() {
             return;
         }
 
-        println!("{}", game);
+        eprintln!("{}", game);
 
-        let opp_valid_moves = game.all_valid_moves_for_color(opp_color);
-        print_move_list(&opp_valid_moves);
+        // let opp_valid_moves = game.all_valid_moves_for_color(opp_color);
```
Comment (vdoubleu) [inline src/main.rs:56]: instead of commenting this out, couldnt we make this `eprintln` as well?

File: src/main.rs [hunk 3]
```diff
@@ -66,7 +69,10 @@ fn main() {
                 }
 
                 let m = match generate_move(&game, your_color) {
-                    Some(m) => m,
+                    Some(m) => {
+                        println!("{}", m);
```
Comment (vdoubleu) [inline src/main.rs:73]: ditto from above

File: src/main.rs [hunk 4]
```diff
@@ -80,10 +86,10 @@ fn main() {
                     return;
                 }
 
-                println!("{}", game);
+                eprintln!("{}", game);
 
-                let opp_valid_moves = game.all_valid_moves_for_color(opp_color);
-                print_move_list(&opp_valid_moves);
+                // let opp_valid_moves = game.all_valid_moves_for_color(opp_color);
```
Comment (vdoubleu) [inline src/main.rs:91]: ditto from above

File: src/main.rs (hunk 1 above)
Comment (David-Cha) [reply src/main.rs:39]: Needed for both the prof's script and my script, I think there is also one more requirement for the prof's script to do later

File: src/main.rs (hunk 3 above)
Comment (David-Cha) [reply src/main.rs:73]: Same comment as above, program needs to output move to stdin

**Pull request merged**